qiskit/transpiler/passes/cx_cancellation.py

```diff
@@ -29,8 +29,10 @@ def run(self, dag):
             chunk = []
             for i in range(len(cx_run) - 1):
                 chunk.append(cx_run[i])
-                qargs0 = dag.node(cx_run[i])["qargs"]
-                qargs1 = dag.node(cx_run[i + 1])["qargs"]
+
+                qargs0 = cx_run[i]["qargs"]
+                qargs1 = cx_run[i + 1]["qargs"]
```

Comment:
can we make these `.qargs` for consistency? we should try to remove dict lookup everywhere.